mod-audit-server/src/main/resources/templates/db_scripts/schema.json

```diff
@@ -74,6 +74,11 @@
       "run": "after",
       "snippetPath": "acquisition/create_acquisition_order_line_log_table.sql",
       "fromModuleVersion": "mod-audit-2.7.0"
+    },
+    {
+      "run": "after",
+      "snippetPath": "acquisition/create_acquisition_piece_log_table.sql",
+      "fromModuleVersion": "mod-audit-2.8.1"
```

Comment:
should be next major release version - 2.9.0
And this version also should be updated to 2.9.0-SNAPHSOT https://github.com/folio-org/mod-audit/blob/master/pom.xml#L6

Reply:
Changed